fenix/app/src/main/res/layout/component_tabstray3.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
`component_tabstray3.xml` - this is how you know it's actually tabs tray 3.0 and not 2.0. 😄